### ISSUE TITLE
Remove Documents from onrr.gov 

### ISSUE THREAD

**Issue by cmathers1** [ONRR/onrr.gov-site#935]
Good morning, 

Please remove the attached documents from onrr.gov 
[950327.pdf](https://github.com/ONRR/onrr.gov-site/files/7000659/950327.pdf)
[880623.pdf](https://github.com/ONRR/onrr.gov-site/files/7000660/880623.pdf)
[930618.pdf](https://github.com/ONRR/onrr.gov-site/files/7000661/930618.pdf)
[20010315.pdf](https://github.com/ONRR/onrr.gov-site/files/7000662/20010315.pdf)
[930111.pdf](https://github.com/ONRR/onrr.gov-site/files/7000663/930111.pdf)
[20010529.pdf](https://github.com/ONRR/onrr.gov-site/files/7000664/20010529.pdf)
[20110829 (1).pdf](https://github.com/ONRR/onrr.gov-site/files/7000665/20110829.1.pdf)
Thank you, 

Christina

**Comment by lpgoldstein** [comment]
@cmathers1 I've removed the documents from the dev site, please review: http://2017onrr/DearRep.htm

**Comment by cmathers1** [comment]
Thank you.  I reviewed and didn't see the documents. Thank you.

Sincerely,

Christina

Christina Mathers
Staff Assistant
Director/Deputy Director's Office
Office of Natural Resources Revenue
6th Avenue and Kipling Street Bldg. 85
Denver, Colorado 80225
Phone:  303-231-3429
Cell: 303-503-1315
________________________________
From: lpgoldstein ***@***.***>
Sent: Wednesday, August 18, 2021 4:08 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Mathers, Christina ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Remove Documents from onrr.gov (#935)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@cmathers1<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcmathers1&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ceafee7e7061a4bdea79f08d9623022da%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648781154980435%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=UlD%2Fdq47khLBMZ6gHjr5sRBu9e7TKN%2BTDkL3qlQfGbg%3D&reserved=0> I've removed the documents from the dev site, please review: http://2017onrr/DearRep.htm

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F935%23issuecomment-900989711&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ceafee7e7061a4bdea79f08d9623022da%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648781154990392%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BZOThLxQnqXHpjTR2%2FTwW6GCAmXubaXcOz%2Fxva8kylw%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAU5GF7SQWQD4TL2RWU4TAA3T5OBBJANCNFSM5CJ7HAQQ&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ceafee7e7061a4bdea79f08d9623022da%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648781155000349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8hABHAi6H01ErOoJuRaATvk%2Ft661ldvOdloZpgkqSDg%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ceafee7e7061a4bdea79f08d9623022da%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648781155000349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4y%2FP0Vxa2pftEmZajzinQ2wOedUvrGr7QHZtw6186HE%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Ceafee7e7061a4bdea79f08d9623022da%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637648781155000349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BYVGdtJe601EadxMqth0%2BhgrCn581FY7Mg1NM9hahls%3D&reserved=0>.

**Comment by lpgoldstein** [comment]
@mcharg will you please spot check just in case?

**Comment by cmathers1** [comment]
I did a search by number and subject, and I couldn't pull any of them up.  I have one more payor/reporter letter on the spreadsheet to check on to see if it still needs to be on onrr.gov. Thanks!



Sincerely,

Christina

Christina Mathers
Staff Assistant
Director/Deputy Director's Office
Office of Natural Resources Revenue
6th Avenue and Kipling Street Bldg. 85
Denver, Colorado 80225
Phone:  303-231-3429
Cell: 303-503-1315
________________________________
From: lpgoldstein ***@***.***>
Sent: Wednesday, August 18, 2021 10:31 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Mathers, Christina ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Remove Documents from onrr.gov (#935)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@mcharg<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmcharg&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Cb8bb0656262e4064bfb308d96265b63a%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649011274947678%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=DoeUc65T0O7F1gm%2BdXIxhCKedxUYZZucTqsPoAcXFv8%3D&reserved=0> will you please spot check just in case?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F935%23issuecomment-901257600&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Cb8bb0656262e4064bfb308d96265b63a%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649011274947678%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ueTepeoIFmJ%2BIW3tkv%2Fx2lv3rkqPnNo2NZowKvLOdXs%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAU5GF7VMPA7XCGCJVQXTMP3T5PN63ANCNFSM5CJ7HAQQ&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Cb8bb0656262e4064bfb308d96265b63a%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649011274957633%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NAFLgTEErSeu4e28OuZTKDe9DrV47zvssjeWOlyxFkY%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Cb8bb0656262e4064bfb308d96265b63a%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649011274957633%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=dLL3MoouL5DTEYOdLWgYp%2FepFIRdGZKB3ek7uNZP5w4%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cchristina.mathers%40onrr.gov%7Cb8bb0656262e4064bfb308d96265b63a%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637649011274967590%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=sOS%2BIR59zH7HMwfDt9aQR2ch2%2Fu1A18bfd3yHN6xtV8%3D&reserved=0>.

**Comment by mcharg** [comment]
I'm not seeing any of them in the list in dev.

**Comment by lpgoldstein** [comment]
@cmathers1 These changes should be live
